lib/anyplayer/selector.rb

```diff
@@ -60,7 +58,7 @@ def player_load(player)
       log "#{player}:"
       begin
         require "anyplayer/players/#{player}"
-        log "  loaded"
+        log '  loaded'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.